packages/storage-plus/README.md

```diff
@@ -188,6 +188,32 @@ fn demo() -> StdResult<()> {
 }
 ```
 
+### Key types
+
+A `Map` key can be anything that implements the `PrimaryKey` trait. There are a series of implementations of
+`PrimaryKey` already provided (see `packages/storage-plus/src/keys.rs`):
+
+ - `impl<'a> PrimaryKey<'a> for &'a [u8]`
+ - `impl<'a> PrimaryKey<'a> for &'a str`
+ - `impl<'a> PrimaryKey<'a> for Vec<u8>`
+ - `impl<'a> PrimaryKey<'a> for String`
+ - `impl<'a> PrimaryKey<'a> for Addr`
+ - `impl<'a> PrimaryKey<'a> for &'a Addr`
+ - `impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a>> PrimaryKey<'a> for (T, U)`
+ - `impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a> + Prefixer<'a>, V: PrimaryKey<'a>> PrimaryKey<'a> for (T, U, V)`
+ - `impl<'a, T: Endian + Clone> PrimaryKey<'a> for IntKey<T>`
+
+That means that byte and string slices, byte vectors, and strings, can be conveniently used as keys.
+Moreover, some other types can be used as well, like addresses and addresses references, tuples and triples, and
+integer types.
+
+We suggest using `Addr` for keys in storage, instead of `String` or string slices. This implies doing address validation
```

Comment:
I would prefix this a bit:

"If the key represents and address, we suggest using &Addr..."

as I have used eg. escrow ID (string) as a key, where Addr wouldn't make sense.

also, generally &Addr is cheaper unless you need the other for eg. secondary indexes